nls/kappnav.properties

```diff
@@ -88,6 +88,12 @@ table.header.component = Component
 table.header.age = Age
 table.header.console = Console
 
+# Messages for when the resource table is empty 
+table.empty = There are no {0} here. Go add one with 
+table.empty.applications = applications
+table.empty.wasNdCells = WAS ND cells
```

Comment:
We cannot reference WAS ND Cells or Liberty collectives in kappnav. These need to move into prism. (And they might already be there)

Reply:
I'll remove `table.empty.wasNdCells = WAS ND cells` and 
`table.empty.libertyCollectives = Liberty collectives` from the kappnav messages file and make sure they're in the correct section in the prism message file.